astro/src/content/blog/javascript-sdks.mdx

```diff
@@ -0,0 +1,177 @@
+
+This is the recommended approach for two reasons:
+
+* more extensive functionality
+* increased security
+
+Let's discuss each of these in turn.
+
+### Functionality
+
+When you use the Authorization Code grant for authentication, as well as redirects for registration and logout, you are leveraging FusionAuth provided functionality. Your application immediately supports a large number of authentication workflows. These include, but are not limited to:
+
+* forgot password
+* email verification
+* MFA
+* magic link authentication
+* password expiration
+* enterprise single sign-on (SAML, OIDC)
+* social sign-on (Google, Facebook, etc)
+* profile management (with a paid license)
+* passkeys (with a paid license)
+* and more
+
+Now, it's not entirely effortless. You must configure FusionAuth, but all this functionality is present, tested, and maintained over time. For example, instead of having to figure out how to set up email verification, you configure a customizable email and enable it in FusionAuth, and it works. In addition, browser redirects work with any language, framework or browser. This ensures your application's login page is never a barrier to your users.
+
+### Security
+
+When using a redirect, the only application which sees or holes sensitive user credentials is FusionAuth. Your application code never has access to these, which decreases the attack surface area.
+
+In addition, FusionAuth has built-in security features such as:
+
+* the ability to lock user accounts
+* password rules
+* webhook notifications when a user has failed to login
+* breached password notification (with a paid license)
+* rate limiting of sensitive security related actions (with an Enterprise license)
+* and more
+
+[FusionAuth takes security seriously](/security); it's our full time job. By redirecting to FusionAuth, you don't have to worry about any other JavaScript in your application getting access to sensitive information or credentials.
+
+### Look and Feel
+
+Additional functionality and security are great, but you typically want your login pages to look like your application. Doesn't a browser redirect make that difficult?
+
+Well, no. FusionAuth allows complete control over the look and feel of the hosted login pages, including adding or removing HTML, CSS, images and JavaScript to the pages. Learn more about this functionality, which we call [themes](/docs/v1/tech/themes/), including an [example using Tailwind CSS](/docs/v1/tech/themes/tailwind).
+
+One other common objection to redirection is the ability to send a successfully authenticated user right back to where they started. It's common to allow users to log in from multiple states in your SPA, after all. Luckily, you can use the `state` parameter to deep link from any page in your SPA to FusionAuth and then send the user right back to that page after a successful login. Here's a video displaying that:
+
+<YouTube id="-vx5rdy-mvY" />
```

Comment:
Let's have Brand/Design get a cover on this one too! I don't think we should show any videos without those.

Reply:
Great, will add a task.